packages/fiori/src/UploadCollectionItem.ts

```diff
@@ -289,6 +289,12 @@ class UploadCollectionItem extends ListItem {
 		}
 	}
 
+	_onInputKeyUp(e: KeyboardEvent) {
```

Comment:
Better add

```
if (this.getFocusDomRef()!.matches(":has(:focus-within)")) {
		return;
}
```

in `_onkeyup` method, so we can handle pressing 'delete' key press on any elements inside the item